app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -9,13 +9,13 @@ import { mockCloseContact1, mockCloseContact2, mockCloseContact3 } from '../mock
 const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
 const ASSIGNED_USERS = [ 123234, 512678, 910132 ]
 const testInputValues = [
-  { field: 'first_name', value: 'Anthony', inputType: 'FormControl' },
-  { field: 'last_name', value: 'Stark', inputType: 'FormControl' },
-  { field: 'primary_telephone', value: '1234567890', inputType: 'PhoneInput' },
-  { field: 'email', value: 'tinman@starkindustries.com', inputType: 'FormControl' },
-  { field: 'last_date_of_exposure', value: '2020-05-16', inputType: 'DateInput' },
-  { field: 'assigned_user', value: ASSIGNED_USERS[0], inputType: 'FormControl' },
-  { field: 'notes', value: 'Inevitable', inputType: 'FormControl' },
+  { field: 'first_name', value: 'Anthony' },
+  { field: 'last_name', value: 'Stark' },
+  { field: 'primary_telephone', value: '1234567890' },
+  { field: 'email', value: 'tinman@starkindustries.com' },
```

Comment:
This is a real website. Use `example.com`.

Reply:
Sounds good! Changed.